src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchSoundSettings.java

```diff
@@ -0,0 +1,29 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.features.garden.farming.LaneswitchNotification;
```

Comment:
many inused imports

Reply:
I activated the setting in intellij, do I have to commit from there?